pkg/controllerhelpers/scylla_test.go

```diff
@@ -265,3 +265,103 @@ func TestEnsureNodeConfigCondition(t *testing.T) {
 		})
 	}
 }
+
+func TestIsNodeTunedForContainer(t *testing.T) {
+	t.Parallel()
+
+	tcs := []struct {
+		name        string
+		nodeConfig  *scyllav1alpha1.NodeConfig
+		nodeName    string
+		containerID string
+		tuned       bool
+	}{
+		{
+			name:        "empty nodeConfig status",
```

Comment:
nit: say what's the expected state as well - `empty nodeConfig isn't considered to be tuned`

Reply:
(applies globally)